syntax = "proto3";

package syft.core.node.common.action;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";
import "proto/core/pointer/pointer.proto";

message GetOrSetPropertyAction {
  string path = 1;
  syft.core.pointer.Pointer _self = 2;
  syft.core.common.UID id_at_location = 3;
  syft.core.io.Address address = 4;
  syft.core.pointer.Pointer set_arg = 5;
}